### PR TITLE
Feature/fp 004 create function for db

### PR DESCRIPTION
**Pull request by TesaRocks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,8 @@
 ### This project is developed using TypeScript and Express. Under development it runs Nodemon. Please have a look at the following script:
 
 ```"start": "nodemon --watch './\*\*/_.ts' --exec 'ts-node' src/index.ts"```
+
```
Comment (diegolaciar) [inline README.md:8]: let's talk about the things you documented here. 
I think the purpose of this document other.

File: docker-compose.yml [hunk 2]
```diff
@@ -2,7 +2,8 @@ version: "3.8"
 
 services:
   db:
-    image: mysql:latest
+    image: mysql:5.7.34
+    command: --default-authentication-plugin=mysql_native_password
```
Comment (diegolaciar) [inline docker-compose.yml]: We tried this line..  but now I think is not necessary.  let's try removeing line 6 , command.

File: tricks.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Please have a look at the following tricks for successfully running the app:
```
Comment (diegolaciar) [inline tricks.md:1]: then let's chat about the way you code the markup.

**Review by diegolaciar — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (TesaRocks) [reply docker-compose.yml]: Removed and works as it should. Thanks

**Pull request merged**